internal/models/req_meta_data.go

```diff
@@ -0,0 +1,7 @@
+package models
```

Comment:
I think we can avoid creating this package by defining the `MetaData` struct in the `pdk` package (extism_pdk.go) and calling it `HTTPRequestMeta`.

Reply:
resolved!! 😁